packages/components/src/box-control/index.js

```diff
@@ -55,7 +55,10 @@ export default function BoxControl( {
 	splitOnAxis = false,
 	allowReset = true,
 	resetValues = DEFAULT_VALUES,
+	spacingType,
```

Comment:
Perhaps this should be something a bit more abstract, like `allowNegativeValues`? The `BoxControl` right now is only used for padding, margin & gap, but we don't know how things may evolve in the future so it would make sense to keep it a bit more abstract.